<ui version="4.0" >
 <class>ConnectionWebServicesDialog</class>
 <widget class="QDialog" name="ConnectionWebServicesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WebServices Connection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="m_iconState" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap>:/images/network_disconnected_lan.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="m_labelState" >
     <property name="text" >
      <string>Connection ....</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="m_labelConnection" >
     <property name="text" >
      <string>Send Request :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QProgressBar" name="m_sendProgressBar" >
     <property name="value" >
      <number>24</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Receive Request :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QProgressBar" name="m_recieveProgressBar" >
     <property name="value" >
      <number>24</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Abort</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
